lib/src/util.dart

```diff
@@ -10,29 +10,29 @@
 void ignore(_) {}
 
 /// Create a single-element fixed-length list.
-List<Object> list1(Object v1) => List.filled(1, v1);
+List<Object?> list1(Object? v1) => List.filled(1, v1);
 
 /// Create a two-element fixed-length list.
-List<Object> list2(Object v1, Object v2) => List.filled(2, null)
+List<Object?> list2(Object? v1, Object? v2) => List.filled(2, null)
   ..[0] = v1
   ..[1] = v2;
 
 /// Create a three-element fixed-length list.
-List<Object> list3(Object v1, Object v2, Object v3) => List.filled(3, null)
+List<Object?> list3(Object? v1, Object? v2, Object? v3) => List.filled(3, null)
   ..[0] = v1
   ..[1] = v2
   ..[2] = v3;
 
 /// Create a four-element fixed-length list.
-List<Object> list4(Object v1, Object v2, Object v3, Object v4) =>
+List<Object?> list4(Object? v1, Object? v2, Object? v3, Object? v4) =>
     List.filled(4, null)
```

Comment:
Could possibly optimize these a little by doing:
```dart
  List.filled(4, v1)
    ..[2] = v2
    ..[3] = v3
    ..[4] = v4;
```
and avoid one assignment. (Not a big save, but a save!)./